config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  get '/', to: 'welcome#index'
+get 'login', to: redirect('/auth/google_oauth2'), as: 'login'
+get 'logout', to: 'sessions#destroy', as: 'logout'
+get 'auth/:provider/callback', to: 'sessions#create'
+get 'auth/failure', to: redirect('/')
+get 'home', to: 'home#show'
```

Comment:
This is the new root_path where the link (or button if we so choose) to sign in using google will be, along with any other front end design decisions we make.